CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+
+## [unreleased]
+
+## [0.2.0] - 2022-04-05
+
+### Added
+- Time-traveling - the ability to go forwards & backwards in block height within tests. This allows to test time specific data changing within contracts: https://github.com/near/workspaces-rs/pull/73
+- Credentials created from account/contract creation are now allowed to be stored and specified by users. https://github.com/near/workspaces-rs/pull/98
+- [Unstable] Allow users to compile contract projects within tests without having to manually go through this step. https://github.com/near/workspaces-rs/pull/77
+- Batch transactions or transactions with multiple actions are now possible. https://github.com/near/workspaces-rs/pull/72
+- Sandbox node (nearcore binary) logs are now suppressed and can be re-enabled if desired. https://github.com/near/workspaces-rs/pull/85
+- Results now expose logs, receipts, and transaction outcome values. https://github.com/near/workspaces-rs/pull/70
+- Convenience methods `Worker::view_code`, `Worker::view_latest_block`, `Worker::view_account`, `Account::view_account`, `Contract::view_account`, `Contract::view_code` now available. https://github.com/near/workspaces-rs/pull/82
+- Improve error handling. If a transaction fails, this error will now be apart of the `Result` return initially. https://github.com/near/workspaces-rs/pull/83
+- Added `tracing` logging to internal code and examples. https://github.com/near/workspaces-rs/pull/55 and https://github.com/near/workspaces-rs/pull/75
+- Convenient `CallExecutionDetails::{is_success, is_failure}` for testing outcomes of transactions. https://github.com/near/workspaces-rs/pull/58
+- Added `mainnet_archival` and `testnet_archival`, where `ref-finance` example now uses `mainnet_archival`. https://github.com/near/workspaces-rs/pull/57 and https://github.com/near/workspaces-rs/pull/94
+
+
```

Comment:
Might want to add a changes column that has #99, #101, and #102

Also should include #109 in fixes or in `Changes`

Reply:
I'll move those around